src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -388,4 +388,55 @@ describe("When Premium is available", () => {
     const progressCardIllustration = screen.getByTestId("progress-card-image");
     expect(progressCardIllustration).toBeInTheDocument();
   });
+
+  it("opens and closes an FAQ accordion item", async () => {
+    const user = userEvent.setup();
+    const ComposedDashboard = composeStory(LandingUs, Meta);
+    render(<ComposedDashboard />);
+    const faqQuestion = screen.getByText(
+      "What kinds of websites sell my personal information?",
+    );
+    await user.click(faqQuestion);
+    const faqAnswer = screen.getByText(
+      "Certain websites are in the business of collecting and selling people’s personal information without their consent, which is unfortunately legal in the US.",
+      { exact: false },
+    );
+    expect(faqAnswer).toHaveClass("expanded");
+    await user.click(faqQuestion);
+    expect(faqAnswer).not.toHaveClass("expanded");
```

Comment:
After adding `[aria-expanded]`, it would be even nicer to check for that using `toHaveAttribute()`, to ensure that doesn't regress.

(Though I think that would be on `faqQuestion`, so you might still also want to add a `toBeInTheDocument()` for `faqAnswer`.)